extensions/amp-story/1.0/background.js

```diff
@@ -123,14 +123,20 @@ export class AmpStoryBackground {
 
     setStyle(hidden, 'background-image', 'none');
 
+    // Set css background properties immediately so it will always be overridden
+    // by the image if there is one.
+    setStyle(hidden, 'background', cssBackground);
+
     // Image will be swapped on load.
     whenFresh(imgLoad, () => {
-      setStyle(hidden, 'background-image', url ? `url(${url})` : null);
+      if (!url) {
+        return;
+      }
+
+      setStyle(hidden, 'background-image', `url(${url})`);
```

Comment:
_(Disclaimer: I have 0 context on the desktop backgrounds)_

I was trying to understand why, in the previous implementation, we were running this line of code even when we knew `url` was undefined. I imagine this could be to remove the image set on the inactive background? From the implementation, it looks very intentional (`imgLoad` comes from `maybeLoadImage` when it'd have been easier to write a `if` statement to not run the code).
In that case, we might want to keep this previous behavior. Otherwise, if we already know we don't want to execute the `setStyle` instruction, we should not even call `maybeLoadImage` and `whenFresh`. But I'm just worried about the behavior change :)

Reply:
Ah, what if this happens:

0. Runtime creates two backgrounds
1. page1 sets a background-image to background0
2. user navigates to page2, which sets a background-image to background 1
3. user navigates to page3, but this page doesn't have a background-image. Return statement from above prevents background-image from being reset from background0, and user sees the background-image from page1

Reply:
Yeah.  Now, since I'm setting `background` (which is a superset of `background-image`), I believe this should still work.

Great catch though

Reply:
Apparently it depends on the order of the CSS properties ([cf this](https://user-images.githubusercontent.com/1492044/46505043-cfb1f600-c7fd-11e8-8476-fbe0483b554b.png))
Should we keep the old behavior that sets the `background-image` to null, to be safe?